src/libs/OptionsListUtils.ts

```diff
@@ -185,57 +242,50 @@ function getPersonalDetailsForAccountIDs(accountIDs, personalDetails) {
 
 /**
  * Return true if personal details data is ready, i.e. report list options can be created.
- * @param {Object} personalDetails
- * @returns {Boolean}
  */
-function isPersonalDetailsReady(personalDetails) {
-    return !_.isEmpty(personalDetails) && _.some(_.keys(personalDetails), (key) => personalDetails[key].accountID);
+function isPersonalDetailsReady(personalDetails: OnyxEntry<PersonalDetailsList>): boolean {
+    const personalDetailsKeys = Object.keys(personalDetails ?? {});
+    return personalDetailsKeys.length > 0 && personalDetailsKeys.some((key) => personalDetails?.[Number(key)]?.accountID);
```

Comment:
```suggestion
    return  personalDetailsKeys.some((key) => personalDetails?.[Number(key)]?.accountID);
```
If `personalDetailsKeys` is empty, then `.some()` will return `false`